### ISSUE TITLE
Deleting a contact fails

### ISSUE THREAD

**Issue by bamg-shaun** [mailjet/mailjet-apiv3-nodejs#282]
The [documentation for the delete](https://dev.mailjet.com/email/guides/contact-management/#gdpr-delete-contacts) only shows cURL example, so on testing the DELETE contacts endpoint I discovered the SDK is wrong and needs updating.

I'm using node-mailjet v6.0.5

Example code I've used during testing, which has the same format as other calls on the above link.
`
      const _response = await client
        .delete('contacts', { version: 'v4' })
        .id(contactId)
        .request();
`
The above results in a call to `DELETE/v4/contact/3860384733`. which returns 404.

The reason is that the call should be directed at `DELETE/v4/contacts/3860384733`, notice the additional `s` in this call.

Executing a curl using the `.../contacts/...` url works fine, and with `.../contact/...` does not.

Basically the SDK needs updating to use `contacts`.

**Comment by qqilihq** [comment]
Plans to fix this?
